AppCenter/AppCenterTests/MSLogDBStorageTests.m

```diff
@@ -66,11 +68,60 @@ - (void)testLoadTooManyLogs {
   XCTAssertTrue(moreLogsAvailable);
 }
 
-- (void)testLoadJustEnoughLogs {
+- (void)testLoadJustEnoughNormalLogs {
```

Comment:
Here we skip priority in titles, if that is fine then we can remove priority from titles in the other PR that has very long titles.

Reply:
Good point and +1 for consistency!